media-video/subtitler-yuv/subtitler-yuv-0.6.5.ebuild

```diff
@@ -22,6 +22,7 @@ DEPEND="${RDEPEND}"
 PATCHES=( "${FILESDIR}"/${P}-fix-build-system.patch )
 
 src_configure() {
+	append-flags -fcommon
```

Comment:
always prefer `append-cflags -fcommon` over `append-flags` because the latter is too broad

Reply:
Sure, I'll write it down :+1:

Reply:
Changed all to append-cflags